crates/router/src/core/payouts.rs

```diff
@@ -2084,56 +2083,23 @@ pub async fn create_recipient_disburse_account(
             ) {
                 let connector_mandate_details = HashMap::from([(
                     merchant_connector_id.clone(),
-                    PaymentsMandateReferenceRecord {
-                        connector_mandate_id: connector_payout_id.clone(),  // then what will connector_mandate_id be
-                        payment_method_type: Some(api_enums::PaymentMethodType::foreign_from(
-                            &payout_method_data.clone(),
-                        )),
-                        original_payment_authorized_amount: Some(
-                            payout_data.payouts.amount.get_amount_as_i64(),
-                        ),
-                        original_payment_authorized_currency: Some(
-                            payout_data.payouts.destination_currency,
-                        ),
-                        payment_instrument_id: Some(Secret::new(connector_payout_id.clone())),
+                    PayoutsMandateReferenceRecord {
+                        transfer_method_id: Some(connector_payout_id),
                     },
                 )]);
 
+                let common_connector_mandate = CommonMandateReference {
+                    payments: None, // doubt here, i think i need to fetch it here.
```

Comment:
If this is create operation, it will be None for payments 👍

Reply:
ok